src/js/main.js

```diff
@@ -108,8 +115,34 @@ var SupportKit = Marionette.Object.extend({
                     id: res.appUserId
                 });
 
+                var EventCollection = BaseCollection.extend({
+                    url: urljoin('appusers', res.appUserId, 'event'),
+                    model: Event
+                });
+
+                var RuleCollection = Backbone.Collection.extend({
+                    model: Rule
+                });
+
+                // this._events overrides some internals for event bindings in Backbone
+                this._eventCollection = new EventCollection(res.events, {
```

Comment:
Looking ahead to the app user story, it might be easier to track this inside of a user object or controller, unless its OK to simply re-write this var with every init() call.